Atari-Game-Player/materializer/replay_buffer_materializer.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Type, Union, List
+import pickle
+import os 
+DEFAULT_FILENAME = "PyEnvironment"
+from zenml.io import fileio
+from zenml.materializers.base_materializer import BaseMaterializer
+from DQN.model import ReplayBuffer
+
+class ReplayBufferMaterializer(BaseMaterializer):
+    ASSOCIATED_TYPES = [ReplayBuffer]
+
+    def handle_input(
+        self, data_type: Type[Any]
+    ) -> Union[ReplayBuffer, ReplayBuffer]:
+        """Reads a RelayBuffer from a pickle file."""
+
+        super().handle_input(data_type)

```

Comment:
As all of these materializers are basically the same, we should potentially just make them one big one with three different ASSOCIATED_TYPES defined in line 10 here e.g.

Reply:
Sure